servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -2201,6 +2201,19 @@ public void deleteDomainRoleMember(ResourceContext ctx, String domainName, Strin
         setRequestDomain(ctx, domainName);
         memberName = memberName.toLowerCase();
 
+        // if this is the admin role then we need to make sure
+        // the admin is not himself who happens to be the last
+        // member in the role
+        AthenzDomain domain = getAthenzDomain(domainName, false);
+        Role adminRole = getRoleFromDomain(ZMSConsts.ADMIN_ROLE_NAME, domain);
+        if (adminRole == null) {
+            throw ZMSUtils.notFoundError("Invalid domain name specified", caller);
+        }
+        List<RoleMember> members = adminRole.getRoleMembers();
+        if (members.size() == 1 && members.get(0).getMemberName().equals(memberName)) {
+          throw ZMSUtils.forbiddenError("deleteDomainRoleMember: Cannot delete last member of 'admin' role", caller);
```

Comment:
just a minor comment but please use 4 spaces for the indent to be consistent.

Reply:
fixed